features/cellular/framework/targets/UBLOX/AT/UBLOX_AT_CellularStack.cpp

```diff
@@ -430,10 +430,12 @@ UBLOX_AT_CellularStack::CellularSocket *UBLOX_AT_CellularStack::find_socket(int
 void UBLOX_AT_CellularStack::clear_socket(CellularSocket *socket)
 {
     if (socket != NULL) {
-        socket->id       = SOCKET_UNUSED;
```

Comment:
Looks good but what about rx_avail usage ?

Reply:
Will generate separate PR for this today.